test/integration/envvars.bats

```diff
@@ -11,7 +11,7 @@ load helper
 
 @test "errors with non-existant env var using .Env" {
   gomplate -i '{{.Env.FOO}}'
-  [ "$status" -eq 2 ]
+  [ "$status" -eq 1 ]
```

Comment:
I'm thinking this is because `RunTemplate`'s signature has changed... I'd rather it stay the same to not break compatibility.